clang-tools-extra/clangd/Config.h

```diff
@@ -114,6 +114,7 @@ struct Config {
     /// these regexes.
     struct {
       std::vector<std::function<bool(llvm::StringRef)>> IgnoreHeader;
+      bool AnalyzeSystemHeaders = false;
```

Comment:
can you move the comment about regexes near  `IgnoreHeader`.

Also I think `AnalyzeAngledIncludes` would be a more fitting name, WDYT?